docs/ReleaseNotes.md

```diff
@@ -1,6 +1,7 @@
 # Release Notes
 
 ## Difference between the latest release and master
+- Add /simulator/context topic (openscenario_interpreter_msgs/msg/Context type) and enable visualize scenario execution status.
```

Comment:
This feature is already added at v0.1.1.
And visualization of topic `/simulation/context` is not scenario_simulator_v2's feature. It's ScenarioPlayer's feature.